volttrontesting/fixtures/volttron_platform_fixtures.py

```diff
@@ -250,7 +250,7 @@ def volttron_instance_rmq(request):
                 params=[
                     dict(messagebus='zmq', ssl_auth=False),
                     pytest.param(dict(messagebus='zmq', ssl_auth=True), marks=ci_skipif),
-                    pytest.param(dict(messagebus='rmq', ssl_auth=True), marks=rmq_skipif),
+                    #pytest.param(dict(messagebus='rmq', ssl_auth=True), marks=rmq_skipif),
```

Comment:
This needs to be runable...unless you were given other instructions.

Reply:
Chandrika and Shwetha told me to comment that out while they work on the RMQ issue.